index.html

```diff
@@ -277,13 +277,46 @@ <h2 class="title-block__name _title">Work experience</h2>
                   >
                 </div>
                 <div class="work-card__main">
+                  <h2 class="work-card__category">Info</h2>
                   <p class="work-card__text">
-                    The company is developing a second version of the product. I
-                    am supporting 2 projects: SPA UI and Business logic/API for
-                    working with data <br />Responsibilities: Interface
-                    development, adding new functionality, supporting legacy
-                    functionality, writing high-load database queries.
+                    The team is develops online surveys with complex
+                    functionality. Project contains SPA UI and Business
+                    logic/API for working with data.
                   </p>
+                  <h2 class="work-card__category">Responsibilities</h2>
+                  <ul class="work-card__list">
```

Comment:
Well, you added `work-card__list` for html, but not for css. I think if `<ul>` don't need class attribute, let's delete it